content/pt-br/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -0,0 +1,713 @@
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  creationTimestamp: 2016-02-18T18:54:22Z
+  name: game-config-3
+  namespace: default
+  resourceVersion: "530"
+  uid: 05f8da22-d671-11e5-8cd0-68f728db1985
+data:
+  game-special-key: |
+    enemies=aliens
+    lives=3
+    enemies.cheat=true
+    enemies.cheat.level=noGoodRotten
+    secret.code.passphrase=UUDDLRLRBABAS
+    secret.code.allowed=true
+    secret.code.lives=30
+```
+
+#### Criando um mapa de configuração a partir de valores literais
+
+Você pode usar `kubectl create configmap` com o argumento `--from-literal` para definir um valor literal a partir da linha de comando:
+
+```shell
+kubectl create configmap special-config --from-literal=special.how=very --from-literal=special.type=charm
+```
+
+Você pode passar vários pares de chave-valor. Cada par fornecido na linha de comando é representado como uma entrada separada na seção `data` do mapa de configuração.
+
+```shell
+kubectl get configmaps special-config -o yaml
+```
+
+A saída é semelhante a esta:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  creationTimestamp: 2016-02-18T19:14:38Z
+  name: special-config
+  namespace: default
+  resourceVersion: "651"
+  uid: dadce046-d673-11e5-8cd0-68f728db1985
+data:
+  special.how: very
+  special.type: charm
+```
+
+### Crie um mapa de configuração de um gerador
+
+O `kubectl` suporta `kustomization.yaml` desde a versão 1.14.
+Você também pode criar um mapa de configuração a partir de geradores, e então aplicá-lo na criação de objetos no `Apiserver`. 
+Os geradores devem ser especificado em um `kustomization.yaml` dentro de um diretório.
+
+#### Gere o mapa de configuração a partir de arquivos
+
+Por exemplo, para gerar um mapa de configuração a partir de arquivos `configure-pod-container/configmap/game.properties`
+
+```shell
+# Criando um arquivo kustomization.yaml com o `ConfigMapGenerator`
+cat <<EOF >./kustomization.yaml
+configMapGenerator:
+- name: game-config-4
+  files:
+  - configure-pod-container/configmap/game.properties
+EOF
+```
+
+Aplique o diretório de kustomization para criar o objeto mapa de configuração.
+
+```shell
+kubectl apply -k .
+configmap/game-config-4-m9dm2f92bt created
+```
+
+Você pode verificar se o mapa de configuração foi criado, assim:
+
+```shell
+kubectl get configmap
+NAME                       DATA   AGE
+game-config-4-m9dm2f92bt   1      37s
+
+
+kubectl describe configmaps/game-config-4-m9dm2f92bt
+Name:         game-config-4-m9dm2f92bt
+Namespace:    default
+Labels:       <none>
+Annotations:  kubectl.kubernetes.io/last-applied-configuration:
+                {"apiVersion":"v1","data":{"game.properties":"enemies=aliens\nlives=3\nenemies.cheat=true\nenemies.cheat.level=noGoodRotten\nsecret.code.p...
+
+Data
+====
+game.properties:
+----
+enemies=aliens
+lives=3
+enemies.cheat=true
+enemies.cheat.level=noGoodRotten
+secret.code.passphrase=UUDDLRLRBABAS
+secret.code.allowed=true
+secret.code.lives=30
+Events:  <none>
```

Comment:
Same comment as before applies here. Command and output should be broken down into two separate fenced code blocks, first one of type ` ```shell `, second one of empty type (` ``` `). There's also a missing connecting sentence prior to these two blocks.